assets/js/googlesitekit/widgets/components/Widget.js

```diff
@@ -27,8 +27,8 @@ const Widget = ( {
 	className,
 	slug,
 	noPadding,
-	header: Header,
-	footer: Footer,
+	Header,
+	Footer,
```

Comment:
PropTypes need to be updated accordingly here. Note that the type also needs updating, not just the case.

That is, these should both expect a `PropTypes.elementType` now instead of an `element`.